pygerm/caproto.py

```diff
@@ -70,10 +69,10 @@ def _path_channel_to_Path(chanel, string_encoding='latin-1'):
         now = datetime.datetime.now()
         write_subdir = Path(f"{now.year}/{now.month}/{now.day}")
         write_subdir = filepath / write_subdir
-        #parent.filepath_channel.write(str(write_subdir))
+        # parent.filepath_channel.write(str(write_subdir))
 
-        #if write_path.is_absolute():
-            #raise Exception("write path must be not absolute")
+        # if write_path.is_absolute():
+        # raise Exception("write path must be not absolute")
```

Comment:
A minor thing, but this whitespace change broke the semantic indentation whitespace.